templates/components/engines/typescript/agent/tools/index.ts

```diff
@@ -28,8 +27,6 @@ export async function createTools(toolConfig: {
 }): Promise<BaseToolWithCall[]> {
   // add local tools from the 'tools' folder (if configured)
   const tools = await createLocalTools(toolConfig.local);
-  // add tools from LlamaIndexTS (if configured)
```

Comment:
If we remove llamahub we also need to remove it from the generator for the tools.json and for the python code